docs/configurations/index.rst

```diff
@@ -8,7 +8,7 @@ and how to create new ones.
 
 OCIO Configurations can be downloaded here: `.zip <http://github.com/imageworks/OpenColorIO-Configs/zipball/master>`__ `.tar.gz <http://github.com/imageworks/OpenColorIO-Configs/tarball/master>`__ (OCIO v1.0+)
 
-If you are interested in crafting custom color configurations, and need assistance, please contact: `ocio-dev <http://groups.google.com/group/ocio-dev>`__\.
+If you are interested in crafting custom color configurations, and need assistance, please contact: `ocio-dev <https://lists.aswf.io/g/ocio-dev>`__\.
```

Comment:
This could probably be debated either way, but to me, people creating configs would be better considered as "users" and so the better list would be ocio-user.  The ocio-dev list seems like it is more for people developing the library itself.  Not a strong objection on my part, just wanted to raise the question.

Reply:
Oh, and I just noticed in the next file it points people creating configs to ocio-user, so this convinces me even more that this one should point to ocio-user.

Reply:
Good call. I will change this.